Src/GoogleApis.Auth.DotNet4/GoogleApis.Auth.DotNet4.csproj

```diff
@@ -33,6 +33,9 @@
     <DocumentationFile>Google.Apis.Auth.PlatformServices.xml</DocumentationFile>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="BouncyCastle.Crypto">

```

Comment:
As you can see here https://www.nuget.org/packages/Google.Apis.Auth/

Google.Apis.Auth package depends on Google.Apis.Core. Because you are bringing bouncy dependency, you need to add it to nuget packages specification as well, otherwise the resulting Google.Apis.Auth package will be broken. What "other" nuget packages did you have in mind?